docs/install_guide.md

```diff
@@ -0,0 +1,14 @@
+## Supported Platforms
```

Comment:
Maybe this page name should be "supported_platforms.md", and it should be the first bullet under "Install Guide" in README.md? I'm not sure what, if anything belongs in a top level "Install Guide" file

Reply:
Moved to "supported_platforms.md".